ansible/roles/common/tasks/main.yml

```diff
@@ -21,6 +21,13 @@
   retries: 10
   delay: 15
   until: epelresult is not failed
+# Update ca-certificates to ensure that CA certificates are new enough
+# to support SSL verification against websites used to download software
+- name: "update ca-certificates package to latest version"
+  yum:
+    name: 'ca-certificates*'
```

Comment:
What do the package names look like?  Wondering if this is the most specific the package pattern can be.

Reply:
> On CentOS 7.5, ca-certificates appears like it would be enough.

Since Muchos is targeting centos, seems like if it could be more narrow that would be good. 

> Note that it will only update and not install any new packages if the wildcard matches multiple.

I did not realize that.  Seems like that makes it pretty safe for any cases I can think of, but with something like this I would err on the side of caution.

Reply:
> > Note that it will only update and not install any new packages if the wildcard matches multiple.
> 
> I did not realize that. Seems like that makes it pretty safe for any cases I can think of, but with something like this I would err on the side of caution.

The [update_only](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#parameter-update_only) option to the yum module is what ensures it will only update installed packages matching the pattern to the latest version.